microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -132,6 +132,46 @@ def verify_dpdk_ovs(
         finally:
             ovs.stop_ovs()
 
+    @TestCaseMetadata(
+        description="""
+           Build and run DPDK multiprocess client/server sample application
+        """,
+        priority=3,
+        requirement=simple_requirement(
+            min_nic_count=2,
+            network_interface=Sriov(),
+        ),
+    )
+    def verify_dpdk_multiprocess(
+        self, node: Node, log: Logger, variables: Dict[str, Any]
+    ) -> None:
+        pmd = "failsafe"
+        # initialize DPDK with sample applications selected for build
+        test_kit = initialize_node_resources(
+            node,
+            log,
+            variables,
+            pmd,
+            sample_apps=[
+                "multi_process/client_server_mp/mp_server",
+                "multi_process/client_server_mp/mp_client",
+            ],
+        )
+
+        # enable hugepages needed for dpdk EAL
+        _init_hugepages(node)
+
+        # setup and run mp_server application
+        examples_path = test_kit.testpmd.dpdk_build_path.joinpath("examples")
+        server_app_path = examples_path.joinpath("dpdk-mp_server")
+        kit_cmd_pair = {test_kit: f"{server_app_path} --help"}
+        output: Dict[DpdkTestResources, str] = dict()
+        task_manager = _start_testpmd_concurrent(kit_cmd_pair, 5, log, output)
+
+        # TODO: Just enough to test the changes, rest of test incoming.
```

Comment:
Is it a draft PR?

Reply:
I was hoping to check the initial refactors in before adding the rest of the test logic, so the test is sort of just a stub to verify that everything builds and runs.